lib/carlo.js

```diff
@@ -517,4 +519,68 @@ function enterTestMode() {
   testMode = true;
 }
 
+async function downloadChromium() {
```

Comment:
Let's move this all into find_chrome.js.

Reply:
done.